manager/manager.py

```diff
@@ -249,31 +249,23 @@ def start_job(self, job_id):
                     if error.startswith('Bad Request: '):
                         error = error[len('Bad Request: '):]
                     job.error_message(error)
-
-            if 'Payload' in json_data:
-                payload = json_data['Payload']
-
-                print('Payload:')
-                print(json.dumps(payload))
-
-                for message in payload['log']:
-                    if message:
-                        job.log_message(message)
-                for message in payload['errors']:
-                    if message:
-                        job.error_message(message)
-                for message in payload['warnings']:
-                    if message:
-                        job.warning_message(message)
-
-                success = payload['success']
-
-                if payload['errors']:
-                    job.log_message('{0} function returned with errors.'.format(module.name))
-                elif payload['warnings']:
-                    job.log_message('{0} function returned with warnings.'.format(module.name))
-                elif payload['log']:
-                    job.log_message('{0} function returned.'.format(module.name))
+                elif 'success' in json_data:
+                    success = json_data['success']
```

Comment:
A preferred programming practice is to avoid magic strings like 'success' and 'errors'.  It is better to assign the string to a global variable (such as "SUCCESS = 'success' ") and use the variable name in the code.  That way the compiler/IDE will help catch typos and inconsistencies.